setup/3. Importing.md

```diff
@@ -0,0 +1,19 @@
+# Importing your Project
+
+## Importing into IntelliJ
+
+If you followed the `Cloning from your IDE` step in the `Git and Github` tutorial, you can skip ahead to `Launching Minecraft`.
+
+Otherwise, if you downloaded the ZIP or cloned your project with git manually, open IntelliJ.
+A window will open with an option to `Open or Import`. Select this and click on the folder where your project is in. 
+The folder you click on should have files like `build.gradle` and a folder named `src` in it. Do not select the folder above your project 
+
+The above process can also be done if IntelliJ already has a project open, by clicking `File` and then `Open`
+
+IntelliJ will begin importing your project and downloading the files needed to mod. This may take several minutes to an half hour depending on how good your computer is.
```

Comment:
```suggestion
IntelliJ will begin importing your project and downloading the files needed to mod. This may take several minutes to half an hour depending on how good your computer is.
```